app/com/linkedin/drelephant/analysis/AnalyticJobGeneratorHadoop2.java

```diff
@@ -46,10 +46,14 @@
   private static final String IS_RM_HA_ENABLED = "yarn.resourcemanager.ha.enabled";
   private static final String RESOURCE_MANAGER_IDS = "yarn.resourcemanager.ha.rm-ids";
   private static final String RM_NODE_STATE_URL = "http://%s/ws/v1/cluster/info";
+  private static final String FETCH_ONLY_RECENT_RM_APPS = "drelephant.analysis.only.fetchRecentApps";
```

Comment:
Ok yes I agree. Based off the code I assumed the property was being pulled out of the Hadoop configuration files but I now realize I was mistaken. I got mixed up by the fact that the `configuration` object includes `GeneralConf.xml` in addition to Hadoop configuration properties. Perhaps you should add this as a commented parameter in `GeneralConf.xml` so that people know it's there? At some point we should come up with a clearly organized document listing/detailing all of Dr. Elephant's configuration parameters.